addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1039,8 +1039,23 @@ registry.anchorSlide = publicWidget.Widget.extend({
         if (!$anchor.length || !scrollValue) {
             return;
         }
-        ev.preventDefault();
-        this._scrollTo($anchor, scrollValue);
+
+        const offcanvasEl = document.querySelector('#top_menu_collapse_mobile');
```

Comment:
The problem is not as big for the non-mobile off canvas menu but I think it would be easy fixing it the same at the same time here? :+1: Also: did it work in the old off canvas menus in stable? Should not it be fixed in older versions? (I am fine with 17.0 only but just curious if it was somehow different).

Reply:
Here is the 16.0 behavior (with regular menu and long menu): https://drive.google.com/file/d/1SAe2up6J8pKlNDkayx7Qmj7wyHCPW0SK/view?usp=drive_link

Reply:
> It's just that it doesn't close.

Well my point was that it's not: you see it in your video, the menu jumps when you click on that button :grimacing: 

>  Note: in <17, le page scrolls correctly, despite the offcanvas not closing

Then I'd say that if the menu does not jump in desktop over there, I'd fix in 17.0 only